bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -325,7 +347,8 @@ public Response listenAndAnswer(
             @QueryParam("sourceId") @Parameter(description = "source ID") @Nullable String sourceId,
             @QueryParam("sttId") @Parameter(description = "Speech-to-Text ID") @Nullable String sttId,
             @QueryParam("ttsId") @Parameter(description = "Text-to-Speech ID") @Nullable String ttsId,
-            @QueryParam("hliId") @Parameter(description = "interpreter ID") @Nullable String hliId,
+            @QueryParam("hliId") @Parameter(description = "interpreter ID", deprecated = true) @Nullable String hliId,
```

Comment:
@J-N-K : is it really necessary to keep this parameter and mark it as deprecated while this API is not present in OH 3.2 ? It was recently added by me in 3.3.